session.php

```diff
@@ -84,27 +91,89 @@
 $formprefill['sessionid'] = $sessionid;
 $markingform = new \mod_observation\form\pointmarking(null, $formprefill);
 
+// Render page.
+$pageurl = new moodle_url('/mod/observation/session.php', array(
+    'sessionid' => $sessionid, 'pointid' => $pointid,
```

Comment:
with multiline arrays like this, I generally find it cleaner to do 1/line. Maybe try it and see if you prefer it